src/Public/js/myradio.timeslot.js

```diff
@@ -1,3 +1,14 @@
+//Name helper function for concating names;
+function NameHelper(data) {
```

Comment:
maybe a better name for this (`formatName`? or something along those lines)

Reply:
Change name to formatName as that works